django/conf/project_template/project_name/settings.py-tpl

```diff
@@ -10,10 +10,10 @@ For the full list of settings and their values, see
 https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/
 """
 
-import os
+from pathlib import Path
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
You should use `.parents[1]` here.

I'd also use `.resolve(strict=True)` as we're expecting this path to exist, but perhaps that isn't necessary as `__file__` must exist.

Reply:
Thanks for the tip. I've done this on the existing code in #12007.

Reply:
I applied the `.resolve(strict=True)` fix to the existing code in #12008.